GlobalViolence/R/DataPrep_Closeout.R

```diff
@@ -51,7 +51,7 @@ for (i in 1:3){
 	GBDi$ISO3     <- as.character(GBDi$ISO3)
 
 	GBDi %>% split(list(GBDi$location, GBDi$sex, GBDi$year), drop = TRUE) %>% 
-	  mclapply(GBD.closeout, mc.cores = (detectCores() - 1),
+	  lapply(GBD.closeout,
```

Comment:
OK, looks like we should always use `library(parallel)`, but then only conditionally load `library(parallelsugar)`. You can ammend the commit to do this and this PR updates accordingly.

Reply:
yep.